public/server/db/og.json

```
@@ -0,0 +1,3 @@
+{
+  "definition": "Original Gangsters"
```

I think the "s" should go.
```suggestion
  "definition": "Original Gangster"
```